File: Add Code Here/PYTHON/RockPaperScissorGame.py
print("Bye World! : (")

import random

# ASCII Arts for rock, paper, and scissors by Veronica Karlsson

rock = '''  
    _______
---'   ____)  
      (_____)  
      (_____)  
      (____)
---.__(___)  
'''

paper = '''  
    _______
---'   ____)____  
          ______)  
          _______)  
         _______)
---.__________)  
'''

scissors = '''  
    _______
---'   ____)____  
          ______)  
       __________)  ....

      (____)
---.__(___)  
'''

# Adding Game Images into a list  
game_images = [rock, paper, scissors]

# Taking input from user choice  
user_choice = int(input(  
    "What do you choose? Type 0 for Rock, 1 for Paper, 2 for scissor. \n => "))

print("User Choice: ")

# print game image by user choice  
print(game_images[user_choice])

# random computer choice  
computer_choice = random.randint(0, 2)

print(" Computer Choice: ")

# print game image by computer choice  
print(game_images[computer_choice])

# rules in logic  
if user_choice == 0 and computer_choice == 2:  
    print("You win! 🎉")  
elif computer_choice == 0 and user_choice == 2:  
    print("You lose.  ☠")  
elif computer_choice > user_choice:  
    print("You lose. ☠")  
elif user_choice > computer_choice:  
    print("You win!🎉 ")  
elif computer_choice == user_choice:  
    print("It's a draw.")  
elif user_choice >= 3 or user_choice < 0:  
    print("You typed an invalid number, You Lose.  ☠")

File: Add Code Here/Java/Rock-Paper-Scissor-Game.java
import java.util.Random;
import java.util.Scanner;

public class RockPaperScissors {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();

        String[] choices = {"rock", "paper", "scissors"};

        while (true) {
            int computerChoice = random.nextInt(3); // 0 for rock, 1 for paper, 2 for scissors
            System.out.print("Enter your choice (rock, paper, scissors): ");
            String userChoice = scanner.nextLine().toLowerCase();

            if (!userChoice.equals("rock") && !userChoice.equals("paper") && !userChoice.equals("scissors")) {
                System.out.println("Invalid choice. Please enter rock, paper, or scissors.");
                continue;
            }

            System.out.println("Computer chose: " + choices[computerChoice]);
            System.out.println("You chose: " + userChoice);

            if (userChoice.equals(choices[computerChoice])) {
                System.out.println("It's a tie!");
            } else if ((userChoice.equals("rock") && choices[computerChoice].equals("scissors"))
                    || (userChoice.equals("paper") && choices[computerChoice].equals("rock"))
                    || (userChoice.equals("scissors") && choices[computerChoice].equals("paper"))) {
                System.out.println("You win!");
            } else {
                System.out.println("Computer wins!");
            }

            System.out.print("Do you want to play again? (yes/no): ");
            String playAgain = scanner.nextLine().toLowerCase();
            if (!playAgain.equals("yes")) {
                break;
            }
        }

        scanner.close();
    }
}
